controllers/follow.js

```diff
@@ -0,0 +1,90 @@
+import Sequelize from 'sequelize';
+import models from '../models';
+
+const { iLike } = Sequelize.Op;
+const { User, Follow, Profile } = models;
+
+export default {
+  async addFollower(req, res) {
```

Comment:
Please rename this controller to `followUser`

Reply:
On it